cmd/opentelemetry-collector/app/defaults/default_config.go

```diff
@@ -56,17 +56,20 @@ func CollectorConfig(storageType string, zipkinHostPort string, factories config
 		recTypes = append(recTypes, string(v.Type()))
 	}
 	hc := factories.Extensions["health_check"].CreateDefaultConfig()
+	resProcessor := factories.Processors["resource"].CreateDefaultConfig()
```

Comment:
Isn't the same approach going to be used for collector, as with the agent default config, to only use the `resource` processor if labels defined?

Reply:
yes, done